### PR TITLE
Fix uri/url wording

### PR DESCRIPTION
**Pull request by fonurr**

### CODE REVIEW

File: src/coco/barrel/hover/BarrelHoverCreator.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { getBarrelDetailRepresentation } from "../../../common/domain/barrel/uti
 import HoverBuilder from "../../../common/vscode/hover/HoverBuilder";
 import { getConfig } from "../../config/util/configUtil";
 import localization from "../../../localization";
-import { getBarrelAppUrl, getBarrelWebUrl } from "../../../common/domain/openInZeplin/util/zeplinUrls";
+import { getBarrelAppUri, getBarrelWebUri } from "../../../common/domain/openInZeplin/util/zeplinUris";
```
Comment (mertkahyaoglu) [inline src/coco/barrel/hover/BarrelHoverCreator.ts:9]:
```suggestion
import { getBarrelAppUri, getBarrelWebUrl } from "../../../common/domain/openInZeplin/util/zeplinUris";
```

**Review by mertkahyaoglu — CHANGES_REQUESTED** [review]
Web url was correct. Let's just convert app `url`s to `uri`s.

File: src/coco/barrel/hover/BarrelHoverCreator.ts (hunk 1 above)
Comment (fonurr) [reply src/coco/barrel/hover/BarrelHoverCreator.ts:9]: Still correct :) but I will revert it for consistency with our other apps.

**Review by mertkahyaoglu — APPROVED** [review]

**Comment by fonurr** [comment]
Applied: rebase -i -autosquash

**Pull request merged**